OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -248,6 +264,40 @@ static object LoadDecisions(MiniYaml yaml)
 			return ret;
 		}
 
+		// fair warning I wasn't sure how to set this up so if someone can get it setup right show me so it doesn't happen again. :) ~~StormWing0
+		static object LoadUnitAdditionalUnitLimits(MiniYaml yaml)
```

Comment:
These look really weird but I'm not certain at this point how these are supposed to look in the YAML. Could you please write an example setup with limited MCVs based on conyards and limiting, say, the helis on each other?

Reply:
I've got an example on my end, I'll upload the yaml AI I was using to test the code later if wanted but here's the sample you wanted: 

LimitUnitByActors:
	UnitLimit:
		LimitedUnit: mcv
		#CheckAllStopLimiters: true
		StopProductionLimiters:
			fact: 3

It's the same idea for the LimitBuildingByActors as well. :)

Reply:
Is there anyone that has ideas on how to write both these new loading methods better?  I was trying to write it so the new code didn't have to be used in the yaml to load without error but didn't have any examples I could find of loading non-lists from the yaml.